tests/test_czml.py

```diff
@@ -442,6 +442,19 @@ def test_czml_add_trajectory():
 
     assert repr(extractor.packets) == expected_doc
 
+    x = u.Quantity([1.0, 2.0, 3.0], u.m)
```

Comment:
Can we turn `expected_doc` to a `pytest.fixture`, and have these new lines as a new test?

Reply:
Since we are on it, I have a question:
- The marker `@pytest.mark.skipif("czml3" not in sys.modules, reason="requires czml3")` has been repeated on all the tests. I wonder if there is any way to skip the whole test module if `czml3` is not installed?
- Also, there are a couple more `expected_doc` in the test file. Is there a need to create a fixture for each?

Reply:
@astrojuanlu Could you let me know what do you think about the above questions? If needed, these can be done here itself?